schemas/cgTranscripts.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cgTranscriptSchema = new Schema({
+  bitsId: String,
+  date: Date,
+  applicationType: String,
+  status: String,
```

Comment:
Add a comment to explain the purpose of this field and the possible values this field can take (Use an enum, maybe ?).